### ISSUE TITLE
Error log for files even if they are filtered out

### ISSUE THREAD

**Issue by Shridhad** [cenfun/vitest-monocart-coverage#5]
We are using this reporter in the React + Vite application. However, we are getting the following "not found file" errors for SVG files.  

```
not found file /Users/name/projects/web/app/icons/github_dark.svg
not found file /Users/name/projects/web/app/icons/github_light.svg
not found file /Users/name/projects/web/app/icons/google.svg
```

We have a multi-package mono-repo setup and these errors obscure the Vitest logs. We found these errors are only thrown for SVGs that have not been reached from test files.  However, adding the SVG to the monocart filter has no impact,

```js
const coverageOptions = {
  name: 'Vitest Coverage Report',
  reports: [
      'console-summary',
      'json-summary',
  ],
  outputDir: './coverage/vitest',
  all: {
    dir: 'web',
    filter: {
      '**/*.+(ts|tsx)': true,
    },
  },
  filter: {
    '**/*.+(svg|css)': false,
    '**/*.+(spec|stories).+(ts|tsx)': false,
    '**/*.+(ts|tsx)': true,
  },
};

export default coverageOptions;
```

Unfortunately, I could not reproduce it in a separate fresh vite+react app with a bare minimum configuration. Our application build is quite complex with Vite, Vitest, react, typescript, esbuild, and [SVGR](https://www.npmjs.com/package/vite-plugin-svgr) for loading SVGs as react components. 

I believe the error is logged at https://github.com/cenfun/vitest-monocart-coverage/blob/main/lib/index.js#L168, would it be a good idea to log the error only if the file passes the entry filter?

**Comment by cenfun** [comment]
Thanks for reporting this, it should be fixed in vitest-monocart-coverage@2.1.2

**Comment by Shridhad** [comment]
Thank you so much for the quick fix.
